b-lab-org/ember-cli-impact-core
# [Wellness] `ember-test-selectors` - aka "what i did on the plane"

## overview
This adds [ember-test-selectors](https://github.com/simplabs/ember-test-selectors) as a development dependency.

This allows us to add `data-test-*` attributes to anything with the added benefit of stripping out `data-test-*` attributes from the production build.

## how to test
You can `ember s -e production` to see the production build, locally. To QA, verify everything looks ok.

## some details
- When adding to a component, you must use `data-test-<something>="whatever"` - `data-test="whatever"` will *not* work
- Because `data-test-*` attributes are stripped from the production build, we *cannot* use them in styles. I'd love to add a lint rule here but I couldn't find one that I liked or allowed for custom rules.
